Petrockv3Final.py

```diff
@@ -0,0 +1,55 @@
+print("Welcome to the pet tungsten cube simulator you will take care of this tungsten cube and make sure it stays alive and happy")
+name = input("What would you like to name your cube?\n")
+health = 10
+
+happy = 10
+
+hunger = 10
+
+while health > 0:
+    action = (input("What would you like to do?\n1. Feed the cube \n2. Play with the cube \n3. Nothing \n4. Leave\n"))
+    
```

Comment:
```suggestion
    print (f"{name}'s condition - Health: {health} Happiness: {happy} Hunger: {hunger}")
```
I would print the stats here at the beginning of the loop every time